src/components/main/Main.tsx

```diff
@@ -35,8 +36,14 @@ const StyledLink = styled(Link)(() => ({
   display: 'flex',
   alignItems: 'center',
 }));
-const LinkComponent = ({ children }: { children: JSX.Element }) => (
-  <StyledLink to={HOME_PATH}>{children}</StyledLink>
+const LinkComponent = ({ children }: { children: ReactNode }) => (
+  <StyledLink
+    data-umami-event="header-home-link"
+    data-umami-event-context={Context.Builder}
```

Comment:
Maybe we want `event-page` as well?

Reply:
I wonder if we should create some kind of type, and spread. You could have the same type in ui if you need to pass the params. Maybe overengineered?

```
type Type = {
   event: string;
  eventContext?: string
}

const buildUmamiProps = (args: Type) => ({ 
  data-umami-event: args.event, 
  data-umami-event-context: args.eventContext
})
```

Reply:
The page is already send with each event. The UI is however not really designed to show a specific graph for events by page. But you can filter for a specific page and then you will see the exact number of invocations for each event on that page specifically. I don't know if this is enough. Maybe for the moment ?